src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String OPERATION_SUPPLY = "supply";
+
+    private static final String OPERATION_BUY = "buy";
+
+    private static final String OPERATION_RESULT = "result";
+
+    private static final String COMA_SEPARATOR = ",";
+
+    private static final String TEXT_OF_EXCEPTION = "File not found";
+
+    private static final String REGEX = "\\W";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplyAmount = 0;
+        int buyAmount = 0;
+        StringBuilder builder = new StringBuilder();
+
+        readFromFile(fromFileName, supplyAmount, buyAmount, builder);
+        writeStringToFile(toFileName, builder);
+
+    }
+
+    private void writeStringToFile(String toFileName, StringBuilder builder) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            bufferedWriter.write(builder.toString());
+        } catch (IOException e) {
+            throw new RuntimeException(TEXT_OF_EXCEPTION);
+        }
+    }
+
+    private void readFromFile(String fromFileName, int supplyAmount, int buyAmount,
```

Comment:
This methdod named as readFromFile but do 3 different work inside
1) read from file
2) calculate statisticts result
3) create the report message 
This three code blocks should be divided to separate methods with appropriate names. One method should do only one logical work.